pkg/ddl/index.go

```diff
@@ -1267,6 +1267,12 @@ func errorIsRetryable(err error, job *model.Job) bool {
 	if job.ErrorCount+1 >= variable.GetDDLErrorCountLimit() {
 		return false
 	}
+	errMsg := err.Error()
```

Comment:
can this part call isRetryableError?

Reply:
https://github.com/pingcap/tidb/pull/56630/commits/53627b28cb694ee80f95c0a265b05b7c43f0deb3